src/python/pants/option/config.py

```diff
@@ -131,25 +129,15 @@ def update_seed_values(key: str, *, default_dir: str) -> None:
 
         return all_seed_values
 
-    def get(self, section, option, type_=str, default=None):
-        """Retrieves option from the specified section (or 'DEFAULT') and attempts to parse it as
-        type.
+    def get(self, section, option) -> str | None:
```

Comment:
This should probably have type hints too:

```suggestion
    def get(self, section: str, option: str) -> str | None:
```

Reply:
Fine to do in followup